lessons/29-js-arrays/index.js

```diff
@@ -0,0 +1,86 @@
+//lesson 26
+
+const siblings = ["Blair", "Bobby", "Brandon"];
+const parents = ["Ken", "Flo"];
+
+const family = siblings.concat(parents);
+
+console.log(family);
+
+family.push('Neo');
+
+console.log(family);
+
+family.reverse();
+
+console.log(family);
+
+family[1] = "Mom";
+
+console.log(family);
+
+var i;
+for (let i = 0; i < family.length; i++) {
+
+    console.log(family[i]);
+}
+
+var a;
+for (a of family) {
+    console.log(a);
+}
+
+//extra exercise
+//recipe card
+
+var favRecipe = {
+    title: 'Mole',
+    servings: 2,
+    ingredients: ["cinnamon", "cumin", "cocoa"],
+}
+
+console.log(favRecipe.title);
+console.log('serves: ${favRecipe.servings}');

```

Comment:
In order to use a variable inside of a string these would need to be backticks 

```javascript
console.log(`serves: ${favRecipe.servings}`);
```

Reply:
Thank you, fixed.